packages/gestalt-codemods/generic-codemods/README.md

```diff
@@ -0,0 +1,65 @@
+# Gestalt Generic Codemods
```

Comment:
We also have https://github.com/pinterest/gestalt/blob/master/packages/gestalt-codemods/README.md…how should these files co-exist? Should one link to the other? Both link to each other? Merge the two into one? Something else?

Reply:
MERGED, DONE